vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
Okay, I see the problem.

FYI: The rails-assets gems act as a rails engine and have an `app/assets/javascript/...` directory structure, which is then "merged" with the actual rails app. That's why it doesn't show up. And AFAIK it always takes precedence over any other specified assets paths, such as from react-rails.